api/tests/mocha/controllers/login.spec.js

```diff
@@ -699,8 +697,8 @@ describe('login controller', () => {
         chai.expect(auth.isDbAdmin.args[0]).to.deep.equal([userCtx]);
         chai.expect(users.createAdmin.callCount).to.equal(1);
         chai.expect(users.createAdmin.args[0]).to.deep.equal([userCtx]);
-        chai.expect(auth.getUserSettings.callCount).to.equal(2);
-        chai.expect(auth.getUserSettings.args).to.deep.equal([[userCtx], [userCtx]]);
+        chai.expect(auth.getUserSettings.callCount).to.equal(1);
+        chai.expect(auth.getUserSettings.args).to.deep.equal([[userCtx]]);
```

Comment:
These assertions changed because I removed the second call to getUserSettings for retrieving the newly created admin's settings.